JointPositionGoalController:
    type: valkyrie_translator/JointPositionGoalController
    publish_est_robot_state: true
    joints:
      - rightThumbMotorPitch1
    joint_velocity_limit: 0.1 # rad/s
    joint_limits:
      rightThumbMotorPitch1:
        has_position_limits: true
        min_position: 0
        max_position: 1.6
